deploy/crds/wmc.openshift.io_v1alpha1_windowsmachineconfig_cr.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: wmc.openshift.io/v1alpha1
 kind: WindowsMachineConfig
 metadata:
-  name: example-windowsmachineconfig
+  name: instance
 spec:
   # Add fields here
-  size: 3
+  replicas: 1
+  instanceType: "m4.large"
```

Comment:
Do we no longer have the requirement of m5a.large?

Reply:
Good point. I forgot about that and I've reached the stage where the networking fails to come up.

Reply:
This has still not been addressed

Reply:
> I'll get to this when I reach the networking stage.

This is what I mentioned. I want to defer this change till we reach the networking change considering it's not blocking us.

Reply:
Given that you are introducing this here, I don't see why you need to leave it as a TODO for later.

Reply:
I had a discussion with @aravindhp on this and considering this is a low priority task and no-one will pick it up later because of other higher tickets, I will change it.